snippet-extractor/Pipfile

```diff
@@ -0,0 +1,15 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+langchain = "*"
+transformers = ">=4.31.0, ==4.*"
+nltk = ">= 3.8.1, == 3.*"
+msgpack = ">= 1.0.1, == 1.*"
```

Comment:
Hint: This doesn't use `~=` so that it has the same behavior compared to what we are used to from rust.

I.e. `~= 3.8.1` means `>= 3.8.1, == 3.8.*` which isn't quite what we want/

Reply:
Also langchain currently doesn't do any versioning, i.e. with every update they bump a `0.0.xxx` version.